src/masoniteorm/schema/Blueprint.py

```diff
@@ -352,6 +352,26 @@ def unsigned_integer(self, column, nullable=False):
         )
         return self
 
+    def morphs(self, column, nullable=False):
+        """Sets a column to be used in a polymorphic relationship.
+
+        Arguments:
+            column {string} -- The column name.
+
+        Keyword Arguments:
+            nullable {bool} -- Whether the column is nullable. (default: {False})
+
+        Returns:
+            self
+        """
+        self._last_column = self.table.add_column(
+            "{}_id".format(column), "unsigned_integer", nullable=nullable
+        )
```

Comment:
hmm. What about in the case someone does:

```
table.morphs('record').nullable()
```

? We might have to add self._last_column as a list or tuple and then nullable will act on all the elements in the list?

Reply:
I did not thought about that, I'll look into it